### PR TITLE
reverse string

### PR DESCRIPTION
**Pull request by Nikby53**
add implementation and test

### CODE REVIEW

File: strings.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package string_utils
```
Comment (papahmsolo) [inline strings.go:1]: - can we get rid of `utils` in the name?
- let's add a package description

File: strings_test.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+func TestReverse(t *testing.T) {
+	t.Run("success", func(t *testing.T) {
+		word := "Hello"
+		expected := "olleH"
+		actual := Reverse(word)
+		if actual != expected {
+			t.Errorf("Expected %v instead of %v", expected, actual)
+		}
+	})
```
Comment (papahmsolo) [inline strings_test.go:13]: - no need for subtests with `t.Run()` as there's only one test
- let's add more test cases. Try non-ascii strings, some corner cases like empty string

**Review by papahmsolo — CHANGES_REQUESTED** [review]
Looks good! Let's improve it a bit.

Can we add a readme file describing the repo?

File: strings_test.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+		{"Hello", "olleH", "Reverse"},
+		{"", "", "EmptyString"},
+		{"🧡💛", "💛🧡", "non-ASCII"},
+	}
+	for _, e := range tests {
+		t.Run(e.name, func(t *testing.T) {
+			res := Reverse(e.in)
+			if res != e.exp {
+				t.Errorf("Expected %v instead of %v", e.exp, e.in)
+			}
```
Comment (papahmsolo) [inline strings_test.go:20]: - let's use `q` in format string, to highlight spaces
- `e.in` will show the input string of the test case instead of reversed value

**Review by papahmsolo — CHANGES_REQUESTED** [review]
👀

File: strings.go (hunk 1 above)
Comment (Nikby53) [reply strings.go]: done

File: strings_test.go (hunk 2 above)
Comment (Nikby53) [reply strings_test.go]: already done

File: go.mod [hunk 4]
```diff
@@ -0,0 +1,3 @@
+module github.com/Nikby53/string-utils
+
+go 1.16
```
[the remark this answers is not in the document]
Comment (Nikby53) [reply go.mod]: ridded

File: strings_test.go (hunk 3 above)
Comment (Nikby53) [reply strings_test.go]: done

**Review by alextur — APPROVED** [review]
